RELEASE.md

```diff
@@ -0,0 +1,74 @@
+# Versioning and releasing Google Api Compiler
+This document states how we plan to version and release the Google Api Compiler.
+1. How and when to release Google Api Compiler
+2. How to setup development branch for day to day commits.
+
+## Branches
+This Api Compiler repo contains only _one_ branch:
+
+`master` : This has the lastest code into which all the contribution
+(pull requests) goes. From this branch we will create a temporary release branch
+through which we will do the release to the central repository. After the
+release the temporary release branch will be deleted.
+
+## Steps to release a new version
+1. Once every month (ideally last Monday of every month) move the code from
+master branch into the release branch (create it in the same repo).
+
+2. Now in the release branch bump up the version number in the build.gradle by
+   running
+   ```
+   utils/update_gradle_version.sh
```

Comment:
Add leading "./" so person running it doesn't have to have the current dir (".") in their $PATH.

./utils/update_gradle_version.sh